src/utils/aadGroup.ts

```diff
@@ -0,0 +1,59 @@
+import { Group } from "@microsoft/microsoft-graph-types";
+import { AxiosRequestConfig } from "axios";
+import request from "../request";
+import { odata } from "./odata";
+
+const graphResource = 'https://graph.microsoft.com';
+
+const getRequestOptions = (url: string, metadata: 'none' | 'minimal' | 'full'): AxiosRequestConfig => ({
```

Comment:
Since we're using this function in just one place, and always with fixed configuration, let's remove it and move the request option to the corresponding request to keep the code easier to follow